### ISSUE TITLE
Error when compiling Cannot convert from ``char[]' to `unowned uint8[]?'

### ISSUE THREAD

**Issue by Ramblurr** [mh21/simple-term#8]
### make error
```
  VALAC    simple_term_vala.stamp
simple-term.vala:235.33-235.67: error: Argument 1: Cannot convert from `char[]' to `unowned uint8[]?'
            terminal.feed_child(selection_data.get_text().to_utf8());
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
simple-term.vala:237.33-237.81: error: Argument 1: Cannot convert from `char[]' to `unowned uint8[]?'
            terminal.feed_child(convert_uris(selection_data.get_uris()).to_utf8());
                                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
simple-term.vala:88.13-88.31: warning: `Vte.Terminal.spawn_sync' has been deprecated since 0.48
Compilation failed: 2 error(s), 1 warning(s)
make: *** [Makefile:449: simple_term_vala.stamp] Error 1

```

### dependency versions

```
vte291-0.66.1
vte291-devel-0.66.1
libvala-0.54.3
vala-0.54.3
gtk3-devel-3.24.30
gtk3-3.24.30
```

I'm on Fedora 35.

### workaround

No idea if this is a proper fix or not.

```patch
diff --git a/simple-term.vala b/simple-term.vala
index 5f89073..4a15a16 100644
--- a/simple-term.vala
+++ b/simple-term.vala
@@ -232,9 +232,9 @@ class TerminalWindow : Gtk.Window
         // this is the only way to get a usable target list
         Gdk.Atom[] targets = { selection_data.get_target() };
         if (Gtk.targets_include_text(targets)) {
-            terminal.feed_child(selection_data.get_text().to_utf8());
+            terminal.feed_child((uint8[])selection_data.get_text().to_utf8());
         } else if (Gtk.targets_include_uri(targets)) {
-            terminal.feed_child(convert_uris(selection_data.get_uris()).to_utf8());
+            terminal.feed_child((uint8[])convert_uris(selection_data.get_uris()).to_utf8());
         }
         Gtk.drag_finish(context, true, false, time);
     }
```

**Comment by Ramblurr** [comment]
well apparently you can't escape the tick in GH issue titles :shrug:

**Comment by martinpitt** [comment]
I fixed that in PR #9. Can you please try with current main?

**Comment by Ramblurr** [comment]
Works for me now, thanks!
